app/src/main/java/protect/card_locker/CatimaAppCompatActivity.java

```diff
@@ -20,6 +21,15 @@ protected void attachBaseContext(Context base) {
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        if (Build.VERSION.SDK_INT >= 23) {
```

Comment:
Why does the build number matter? If a user updates the app won't they already have the highest version?

Reply:
The build version here is actually related to Android's API level - 23 in this case corresponds to Android 6.0. This check is so that phones that are running on API levels lower than that would not run that code block (since the feature wasn't supported in API 21 & 22, which Catima is also currently supporting - you can check it in the build.gradle file)